src/staking-v3/components/Dapps.vue

```diff
@@ -85,14 +85,11 @@ export default defineComponent({
 
       const value = props.search.toLowerCase();
 
-      const result = dapps
-        .map((dapp) => {
-          const isFoundDapp =
-            dapp.basic.name.toLowerCase().includes(value) ||
-            dapp.basic.shortDescription.toLowerCase().includes(value);
-          return isFoundDapp ? dapp : undefined;
-        })
-        .filter((it) => it !== undefined);
+      const result = dapps.filter(
+        (dapp) =>
+          dapp.basic.name.toLowerCase().includes(value) ||
+          dapp.basic.shortDescription.toLowerCase().includes(value)
+      );
       return result.length > 0 ? result : [];
```

Comment:
? operator is not needed here. Since `filter` function returns [] if no elements found. Just simply `return result;`